src/odemis/driver/ueye.py

```diff
@@ -695,9 +695,14 @@ def __init__(self, name, role, device=None, **kwargs):
             # Set the format
             # TODO: color mode based on the maximum BPP
             # cf is_DeviceFeature( IS_DEVICE_FEATURE_CMD_GET_SUPPORTED_SENSOR_BIT_DEPTHS)
```

Comment:
I tried to run this command but got a `NOT_SUPPORTED` error. Also looked through the manual but could not find another way to get the supported bit depth.